spec/features/user_google_login_spec.rb

```diff
@@ -54,7 +54,7 @@
     context 'when Google responds with "This is Google OAuth."' do
       let(:google_response_content) { 'This is Google OAuth.' }
 
-      around do |spec|
+      before do
```

Comment:
Changing from `around` to `before` is actually significant. Without this change, the test fails. The reason is because we have a global RSpec `before` block that checks which driver to use (depending on metadata), and we need that block to run before this block. When both are `before`s, that happens. When this one was an `around`, I think it was running first, which was breakingly problematic.